numpy/core/include/numpy/npy_3kcompat.h

```diff
@@ -78,6 +78,22 @@ static NPY_INLINE int PyInt_Check(PyObject *op) {
         } while (0)
 #endif
 
+/* introduced in https://github.com/python/cpython/commit/a24107b04c1277e3c1105f98aff5bfa3a98b33a0 */
+#if PY_VERSION_HEX < 0x030800A3
```

Comment:
Are you sure this is necessary? 3.5 docs list this: https://docs.python.org/3.5/c-api/dict.html#c.PyDict_GetItemWithError

Reply:
Ah, nvm. Sorry, this is the String version, not the item version...

Reply:
Maybe better to `Npy` prefix this though? Plus, the correct symbol on new python 3.8 should be without the underscore, so should we just add an alias for that one on the newer Python? I suppose one reason to not use `#ifdef PyDict_GetItem...` its hard to notice when is is not needed it anymore.

Reply:
> Plus, the correct symbol on new python 3.8 should be without the underscore

Can you give me a reference on that?

Reply:
Sorry, you are right :/. Although I guess that means it may change in the future to remove the underscore (but that is easy to adept). But in that case, may as well keep it as is, it will be a simple thing to fix up if python decides on changing this.
I wonder about PyPy, but we can fix it up when it comes up.

Reply:
Good point about PyPI, but should be easy to fix.

Also, if we have a minimum pypi version we probably ought to have CI that runs against it...